src/scripts/cronjobs/emailBreachAlerts.tsx

```diff
@@ -260,20 +267,62 @@ export async function poll(
                 });
 
                 const l10n = getCronjobL10n(sanitizeSubscriberRow(recipient));
-                const subject = l10n.getString("breach-alert-subject");
-
-                await sendEmail(
-                  recipientEmail,
-                  subject,
-                  renderEmail(
-                    <BreachAlertEmail
-                      l10n={l10n}
-                      breach={breachAlert}
-                      breachedEmail={breachedEmail}
-                      utmCampaignId={utmCampaignId}
-                    />,
-                  ),
-                );
+                const enabledFeatureFlags = await getEnabledFeatureFlags({
+                  email: recipient.primary_email,
+                });
+                if (enabledFeatureFlags.includes("BreachEmailRedesign")) {
+                  /**
+                   * Without an active user session, we don't know the user's country. This is
+                   * our best guess based on their locale. At the time of writing, it's only
+                   * used to determine whether to count SSN breaches (which we don't have
+                   * recommendations for outside the US).
+                   */
+                  const assumedCountryCode = getSignupLocaleCountry(recipient);
+
+                  const scanData = await getLatestOnerepScanResults(
```

Comment:
These may be stale, if that's important then `refreshStoredScanResults` can be `await`ed first.

Reply:
OK, added d0a6996f6 for the three cases you called out here. One thing I did notice, though, is that we have a lot of calls to `getLatestOnerepScanResults` across our codebase that do not call this function first (e.g. every step in the guided resolution). I guess that might be fine since all of those go through the dashboard, where I assume it should get refreshed, but... Yeah, cache invalidation is hard.